builder/googlecompute/step_import_os_login_ssh_key.go

```diff
@@ -57,6 +51,17 @@ func (s *StepImportOSLoginSSHKey) Run(ctx context.Context, state multistep.State
 		s.accountEmail = config.account.jwt.Email
 	}
 
+	if s.GCEUserFunc == nil {
+		s.GCEUserFunc = getGCEUser
+	}
+	if s.accountEmail == "" {
+		s.accountEmail = s.GCEUserFunc()
+	}
```

Comment:
👋🏼 
Is there a case when `s.GCEUserFunc` is not nil here ? You just created the field and I don't see it be set anywhere else ?

```suggestion
	if s.accountEmail == "" {
		s.accountEmail = getGCEUser()
	}
```

Reply:
Ah, it's for tests !